Resources/doc/install/combined_with_fosuser.md

```diff
@@ -0,0 +1,144 @@
+Combined install with FOSUserBundle
+==========
```

Comment:
Shame on me. I should read the documentation next time when I write Markdown text the first time.